internal/cmd/operator-sdk/pkgManToBundle/cmd.go

```diff
@@ -0,0 +1,294 @@
+need to be in the 'PATH' or fully qualified path name should be provided.
+`
+
+	examples = `
+
+# Provide the packagemanifests directory as input to the command. Consider the packagemanifests directory to have the following
+# structure:
+
+$ tree packagemanifests/
+packagemanifests
+└── etcd
+    ├── 0.0.1
+    │   ├── etcdcluster.crd.yaml
+    │   └── etcdoperator.clusterserviceversion.yaml
+    ├── 0.0.2
+    │   ├── etcdbackup.crd.yaml
+    │   ├── etcdcluster.crd.yaml
+    │   ├── etcdoperator.v0.0.2.clusterserviceversion.yaml
+    │   └── etcdrestore.crd.yaml
+    └── etcd.package.yaml
+
+# Run the following command to generate bundles in the default 'bundle/' directory with the base-container image name
+# to be 'quay.io/example/etcd'
+$ operator-sdk pkgman-to-bundle packagemanifests --base-image quay.io/example/etcd
+INFO[0000] packagemanifests will be migrated to bundles in bundle directory
+INFO[0000] Creating bundle/bundle-0.0.1/bundle.Dockerfile
+INFO[0000] Creating bundle/bundle-0.0.1/metadata/annotations.yaml
+...
+
+# After running the above command, the bundles will be generated in 'bundle/' directory.
+$ tree bundle/
+bundle/
+├── bundle-0.0.1
+│   ├── bundle.Dockerfile
+│   ├── manifests
+│   │   ├── etcdcluster.crd.yaml
+│   │   ├── etcdoperator.clusterserviceversion.yaml
+│   ├── metadata
+│   │   └── annotations.yaml
+│   └── tests
+│       └── scorecard
+│           └── config.yaml
+└── bundle-0.0.2
+    ├── bundle.Dockerfile
+    ├── manifests
+    │   ├── etcdbackup.crd.yaml
+    │   ├── etcdcluster.crd.yaml
+    │   ├── etcdoperator.v0.0.2.clusterserviceversion.yaml
+    │   ├── etcdrestore.crd.yaml
+    ├── metadata
+    │   └── annotations.yaml
+    └── tests
+        └── scorecard
+			└── config.yaml
+
+Also, images for the both the bundles will be built with the following names: quay.io/example/etcd:0.0.1 and quay.io/example/etcd:0.0.2.
+`
+)
+
+type pkgManToBundleCmd struct {
+	// Input packagemanifest directory.
+	pkgmanifestDir string
+
+	// Optional flags for generating and building bundles.
+	outputDir string
+	baseImg   string
+	buildCmd  string
+}
+
+// NewCmd returns the pkgManToBundleCmd configured with the provided input options.
+func NewCmd() *cobra.Command {
+	p := pkgManToBundleCmd{}
+
+	pkgManToBundleCmd := &cobra.Command{
+		Use:     "pkgman-to-bundle",
+		Short:   "Migrates packagemanifests to bundles",
+		Long:    longHelp,
+		Example: examples,
+		PreRunE: func(cmd *cobra.Command, args []string) (err error) {
+			return p.validate(args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			p.pkgmanifestDir = args[0]
+			return p.run()
+		},
+	}
+
+	pkgManToBundleCmd.Flags().StringVar(&p.outputDir, "output-dir", "", "Directory to write bundle to.")
+	pkgManToBundleCmd.Flags().StringVar(&p.baseImg, "base-image", "", "Base container image name for bundles.")
```

Comment:
`--image-tag-base` is probably a better name since `base-image` sounds like "what image tag should be used in `FROM` to create my bundle image"
```suggestion
	pkgManToBundleCmd.Flags().StringVar(&p.baseImg, "image-tag-base", "", "Base container image name for bundle image tags, ex. my.reg/foo/bar-operator will become my.reg/foo/bar-operator-bundle:${package-dir-name} for each child directory name in the packagemanifests directory")
```